File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/test/ServiceEndpoint.java
package com.github.kotvertolet.youtubejextractor.models.test;

import com.google.gson.annotations.SerializedName;

public class ServiceEndpoint{

	@SerializedName("feedbackEndpoint")
	private FeedbackEndpoint feedbackEndpoint;

	@SerializedName("commandMetadata")
	private CommandMetadata commandMetadata;

	@SerializedName("clickTrackingParams")
	private String clickTrackingParams;

	public void setFeedbackEndpoint(FeedbackEndpoint feedbackEndpoint){
		this.feedbackEndpoint = feedbackEndpoint;
	}

	public FeedbackEndpoint getFeedbackEndpoint(){
		return feedbackEndpoint;
	}

	public void setCommandMetadata(CommandMetadata commandMetadata){
		this.commandMetadata = commandMetadata;
	}

	public CommandMetadata getCommandMetadata(){
		return commandMetadata;
	}

	public void setClickTrackingParams(String clickTrackingParams){
		this.clickTrackingParams = clickTrackingParams;
	}

	public String getClickTrackingParams(){
		return clickTrackingParams;
	}
}

File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/newModels/ServiceEndpoint.kt
package com.github.kotvertolet.youtubejextractor.models.newModels

import com.google.gson.annotations.SerializedName

class ServiceEndpoint {
    @SerializedName("feedbackEndpoint")
    var feedbackEndpoint: FeedbackEndpoint? = null

    @SerializedName("commandMetadata")
    var commandMetadata: CommandMetadata? = null

    @SerializedName("clickTrackingParams")
    var clickTrackingParams: String? = null
    override fun toString(): String {
        return "ServiceEndpoint{" +
                "feedbackEndpoint = '" + feedbackEndpoint + '\'' +
                ",commandMetadata = '" + commandMetadata + '\'' +
                ",clickTrackingParams = '" + clickTrackingParams + '\'' +
                "}"
    }
}